test/user.test.js

```diff
@@ -1824,25 +1827,20 @@ describe('User', function() {
             User.login(currentEmailCredentials, function(err, accessToken1) {
               if (err) return next(err);
               assert(accessToken1.userId);
+              originalUserToken1 = accessToken1.id;
               next();
             });
           },
           function secondLoginWithOriginalEmail(next) {
             User.login(currentEmailCredentials, function(err, accessToken2) {
               if (err) return next(err);
               assert(accessToken2.userId);
+              originalUserToken2 = accessToken2.id;
               next();
             });
           },
         ], function(err) {
-          done();
-        });
-      });
-
-      afterEach('Deleting original user', function deleteOriginalAccounts(done) {
-        User.deleteById(user.id, function(err, user) {
-          if (err) return done (err);
-          done();
+          done(err);
```

Comment:
I think there is no need to create a dummy function forwarding `err` to `done`. You can write:

```js
], done);
```

Unless I am missing something?

This applies to many other places below too.